muffnn/autoencoder/autoencoder.py

```diff
@@ -38,8 +38,10 @@ class Autoencoder(TFPicklingBase, TransformerMixin, BaseEstimator):
     n_epochs : int, optional
         The number of epochs (iterations through the training data) when
         fitting.
-    dropout : float or None, optional
-        The dropout probability. If None, then dropout will not be used.
+    keep_prob : float, optional
+        The probability of keeping values in dropout. A value of 1.0 means that
+        dropout will not be used. cf. `TensorFlow documentation
+        <https://www.tensorflow.org/versions/r0.11/api_docs/python/nn.html#dropout>`
```

Comment:
please see my response to the comment below and let me know what you think.  if we add those checks for `keep_prob == 1.0` back in, then the current docstring should work, right?